diff --git a/volatile_pintrace/volatile_test.pl b/volatile_pintrace/volatile_test.pl
index 34ba969..e8aed90 100755
--- a/volatile_pintrace/volatile_test.pl
+++ b/volatile_pintrace/volatile_test.pl
@@ -106,10 +106,17 @@ my @ccomp = ("ia32",
              "ccomp",
              \@ccomp_opts);
 
+my @ubuntu_gcc44 = (
+    "ia32",
+    "gcc",
+    "gcc-4.4",
+    \@gcc_opts);
+
 my @compilers_to_test = (
-    \@clang,
-    \@gcccurrent,
-    \@ccomp,
+    \@ubuntu_gcc44,
+    # \@clang,
+    # \@gcccurrent,
+    # \@ccomp,
 );
 
 #########################################################
