diff --git a/utah/scripts/old_john_driver/kill_random.pl b/utah/scripts/old_john_driver/kill_random.pl
index d358e5d..ae6d4fa 100755
--- a/utah/scripts/old_john_driver/kill_random.pl
+++ b/utah/scripts/old_john_driver/kill_random.pl
@@ -2,7 +2,7 @@
 
 use strict;
 
-open INF, "ps axu |" or die;
+open INF, "ps axuwww |" or die;
 while (<INF>) {
     next if (!(/random\_test/));
     my @l = split /\s+/;
